go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/newrelic/go-agent/v3/integrations/nrgrpc v1.3.1
 	github.com/odpf/salt v0.1.1-0.20220903111417-7d83984bc683
 	github.com/pkg/errors v0.9.1
+	github.com/rs/zerolog v1.15.0
```

Comment:
@ravisuhag something is importing this. I think zerolog is not necessary for this right?

Reply:
Yes, fixed. Needed a mod tidy